### DISCUSSION TITLE
usbip: error: tcp connect

### DISCUSSION THREAD

**Discussion by KostiaPerehuda** [dorssel/usbipd-win#157] [Q&A] [answered]
Hi,
I've got this error when trying to run the command:
```
C:\WINDOWS\system32>usbipd wsl attach -b 1-1
[sudo] password for kostia:
usbip: error: tcp connect
usbipd: error: Failed to attach device with ID '1-1'.
```
Any thoughts about this, please?

**Comment by dorssel** [comment]
Probably a duplicate of https://github.com/dorssel/usbipd-win/issues/98#issuecomment-970848471

**Comment by dorssel** [comment, marked as answer]
Since version 2.0.0 there is more detailed reporting on TCP connection/firewall issues. Make sure the host is reachable on TCP port 3240.
